src/language-css/loc.js

```diff
@@ -25,26 +28,68 @@ function calculateLocEnd(node, text) {
 }
 
 function calculateLoc(node, text) {
-  if (node && typeof node === "object") {
-    if (node.source) {
-      node.source.startOffset = calculateLocStart(node, text);
-      node.source.endOffset = calculateLocEnd(node, text);
+  if (node.source) {
+    node.source.startOffset = calculateLocStart(node, text);
+    node.source.endOffset = calculateLocEnd(node, text);
+  }
+
+  for (const key in node) {
+    const child = node[key];
```

Comment:
```suggestion
  for (const [key, child] of Object.entries(node)) {
```

Reply:
I'm not sure about the performance impact of this change.

Reply:
`for...in` is faster.